### PR TITLE
Changes for custom client resource name for SSO

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc [hunk 1]
```diff
@@ -71,6 +78,7 @@ In this example:
 * `resource` is the  name for the client that you created.
 * `credential name` is the  secret key for the client you created. You can find the key in the *Credentials* tab on the *Clients* page of the RH-SSO Admin Console.
 * `principal-attribute` is the login name of the user. If you do not provide this value, your User Id is displayed in the application instead of your user name.
+* `kie` is the client resource name. If you want to use client roles with a different client resource name, replace `kie` with the client resource name, for example, `<resource>customClient</resource>`.
```
Comment (tomasdavidorg) [inline doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc:81]: Shouldn't be this part inside the `resource`? `kie` is not name of the configuration tag but it is possible value of the tag `<resource>`.
```
* `resource` is the name for the client that you created. Optional: If you want to use client roles with a different client resource name, replace `kie` with the client resource name, for example, `<resource>customClient</resource>`.
```

Comment (RishiRajAnand) [reply doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc]: @mramendi @tomas I believe we dont really need this line here"If you want to use client roles with a different client resource name, replace `kie` with the client resource name, for example, `<resource>customClient</resource>`." Since its quite obvious here that client roles are always taken from the client that is requesting keycloak connection this field is rarely changed for client roles purposes . wdyt ?

Comment (mramendi) [reply doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc]: I don't think I can agree. We can't rely on anything being obvious. I did already join the lines as Tomas requested (see current PR) and I do think this explicit explanation should stay, because not everyone has a perfect understanding of RH-SSO.

**Comment by tomas** [comment]
Please don't mention me again in this repository. I have nothing to do with this.

File: doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc (hunk 1 above)
Comment (RishiRajAnand) [reply doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc]: You are right. But by obvious i meant is , Users always have to use the resource name which they have used while configuring the client in RHSSO. They cannot use any client name. A better way to put this would be "If you want to use client roles, use the same client resource name which you have used while configuring client in RHSSO"

**Comment by RishiRajAnand** [comment]
> Please don't mention me again in this repository. I have nothing to do with this.

Apologies, Wont happen again.

File: doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc [hunk 2]
```diff
@@ -30,19 +30,26 @@ NOTE: Install the adapter with the `-Dserver.config=standalone-full.xml` propert
   <property name="org.uberfire.ext.security.management.keycloak.authServer" value="http://localhost:8180/auth"/>
 </system-properties>
 ----
+--
++
+. Optional: If you want to use client roles, also add the following system property:
```
Comment (briandooley) [inline doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc:35]:
```suggestion
. Optional: If you want to use client roles, add the following system property:
```

**Review by briandooley — CHANGES_REQUESTED** [review]
Looks good. Just a small change.

File: doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc (hunk 1 above)
Comment (tomasdavidorg) [reply doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc]: @mramendi @RishiRajAnand I am ok with both.

**Review by tomasdavidorg — APPROVED** [review]

File: doc-content/enterprise-only/integration/sso-central-proc.adoc [hunk 3]
```diff
@@ -10,7 +10,8 @@ This chapter describes how to authenticate {CENTRAL} through RH-SSO. It includes
 .Prerequisites
 * {CENTRAL} is installed in a {EAP} {EAP_VERSION} server, as described in {URL_INSTALLING_AND_CONFIGURING}#assembly-install-on-eap[_{INSTALLING_ON_EAP}_].
 * RH-SSO is installed as described in <<sso-realm-proc>>.
-* {CENTRAL} users have been added to RH-SSO as described in <<sso-user-add-proc>>.
+* You added {CENTRAL} users to RH-SSO as described in <<sso-user-add-proc>>.
+* Optional: In order to be able to manage RH-SSO users from {CENTRAL}, you added all realm-management client roles in RH-SSO to the {CENTRAL} administrator user.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/integration/sso-central-proc.adoc:14]: For minimalism, I would reword as follows:

Optional: In order to manage RH-SSO users from {CENTRAL}, you added all realm-management client roles in RH-SSO to the {CENTRAL} administrator user.

File: doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc [hunk 4]
```diff
@@ -68,7 +75,7 @@ In this example:
 * `realm-public-key` is the public key of the realm you created. You can find the key in the *Keys* tab in the *Realm settings* page of the realm you created in the RH-SSO Admin Console. If you do not provide a value for `realm-public-key`, the server retrieves it automatically.
 * `auth-server-url` is the  URL for the RH-SSO authentication server.
 * `enable-basic-auth` is the  setting to enable basic authentication mechanism, so that the clients can use both token-based and basic authentication approaches to perform the requests.
-* `resource` is the  name for the client that you created.
+* `resource` is the name for the client that you created. If you want to use client roles, use the client resource name that you used when configuring the client in RH-SSO.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc:78]: Same as previous comment:

* `resource` is the name for the client that you created. To use client roles, use the client resource name that you used when configuring the client in RH-SSO.

**Review by michelehaglund — APPROVED** [review]
Minor comments inline. Looks good.
